src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const ageInfo = age && <p className="Person__age">{`I am ${age}`}</p>;
+  const Male = sex === 'm';
+  const partnerSex = Male ? 'wife' : 'husband';
```

Comment:
you can combine these two statements like

``` const partnerSex = sex === 'm'  ? 'wife' : 'husband';```
 
 because you use `Male` only in one place...if we used it somewhere else we would leave it